test/math.jl

```diff
@@ -192,6 +192,25 @@ end
 @test exp2(Float16(2.)) ≈ exp2(2.)
 @test log(e) == 1
 
+@testset "check exp2(::Int) matches exp2(::Float)" begin
+    for ii in -2048:2048
+		expected = exp2(float(ii))
```

Comment:
Thanks, I should have checked that.
Is there a way to use `make check-whitespace` do it?